study-builder/studies/testboston/followup-survey.conf

```diff
@@ -151,6 +151,27 @@
           },
           "blockType": "QUESTION",
           "shownExpr": null
+        },
+        {
+          "question": ${_includes.covid_survey.flu_shot} {
+            "stableId": ${id.q.flu_shot}
+          },
+          "blockType": "QUESTION",
+          "shownExpr": null
+        },
+        {
+          "question": ${_includes.symptoms_survey.sample_collect_date} {
+            "stableId": ${id.q.followup_sample_collect_date}
+          }
+          "blockType": "QUESTION",
+          "shownExpr": null
+        },
+        {
```

Comment:
side effect of making time as separate question is that: 
For users its one question and backend two questions and question counts that we display on the dashboard will be off by 1. 
Trade off: Work around is not hideNumber and Time question will have a number

Reply:
![image](https://user-images.githubusercontent.com/26553609/90486782-1c371e00-e157-11ea-84ef-ab0df7174b6c.png)

Reply:
How about splitting this question into 2 questions?
7. What date did you collect your sample using the home collection kit?
8. What time did you collect your sample using the home collection kit?

To be honest, I don't like it very much and probably should be discussed with Jen

Reply:
I think you can hide just that one question.

Reply:
Yes I can hide it but question count in dashboard shows up as 8 and users don't see Q#8

Reply:
Oh! Yeah, that's an issue...

Reply:
@ssettipalli, could you please send me in the email the questions' JSON, I want to test the styling for these questions.